config/schedule.rb

```diff
@@ -42,6 +42,12 @@
   rake "cleanup:transient_registrations"
 end
 
+# This is the old registration cleanup job which will delete all records, paper trails
```

Comment:
(Pedantic point....) "Call it "the cleanup job for old registrations" - the current wording makes it look like it's the job that's old!

Reply:
done